www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+---
+layout: page
+title: Introducing Karmashapes
+published: false
+date: 2023-06-19
+permalink: /blog/2023/06/19/introducting-karmashapes/
+category: blog
+excerpt: "Thanks to the Karmashapes initiative, Who’s On First now provides the best open data for towns and villages in India."
+authors: [Justin Meyers, stepps00, nvkelso]
+image: "images/coverage_overview.png)"
+tag: [whosonfirst,wof,data,import,india]
+---
+
+[![](images/coverage_overview.png)]()
+
+<div style="font-size:small;font-style:italic;text-align:center;">
+Printing Block (India); wood; 1957-123-6 <a href="https://collection.cooperhewitt.org/objects/18412045/">Collection of Cooper Hewitt Museum</a>.</div>
+
+
+# Improving Locality Coverage in India
+
+Who’s On First now provides the best open data for towns and villages in India, with 915,468 locality records, thanks to the Karmashapes initiative.
+
+![Who's On First India locality coverage](images/coverage_overview.png "Who's On First India locality coverage")
+_(above) Locality coverage of India, visualized. Polygon coverage shown in black, point coverage in blue._
+
+In the Who’s on First gazetteer, a [locality](https://github.com/whosonfirst/whosonfirst-placetypes/tree/main#locality) represents an individual populated place regardless of population or size. While the number of villages in India is often quoted as being closer to 500,000 or 600,000 in total, the number of locality records is higher than this because localities are not limited to certain local government areas. As mentioned in the [State of the Gazetteer](https://whosonfirst.org/blog/2023/06/07/state-of-the-gazetteer/) blog post:
```

Comment:
To: `the number of locality records in Who's On First is significantly larger because WOF localities include both incorporated local government areas and unincorporated settlements`

From: `the number of locality records is higher than this because localities are not limited to certain local government areas`